src/js/helpers/graph/declarationHelper.mock.js

```diff
@@ -0,0 +1,481 @@
+  id: 'A',
+  label: 'A label',
+  status: 'success',
+  type: 'composite',
+  subtasks: [
+    {
+      id: 'A-A',
+      label: 'A-A label',
+      status: 'success',
+      type: 'single',
+      response: {
+        invocations: {},
+      },
+    },
+    {
+      id: 'A-B',
+      label: 'A-B label',
+      status: 'success',
+      type: 'single',
+      response: {
+        invocations: {},
+      },
+    },
+  ],
+  response: {
+    invocations: {},
+  },
+};
+
+export const compositeNodeWithTransitions = {
+  id: 'A',
+  label: 'A label',
+  status: 'success',
+  type: 'composite',
+  subtasks: [
+    {
+      id: 'A-A',
+      label: 'A-A label',
+      status: 'success',
+      type: 'single',
+      response: {
+        invocations: {},
+      },
+    },
+    {
+      id: 'A-B',
+      label: 'A-B label',
+      status: 'success',
+      type: 'single',
+      response: {
+        invocations: {},
+      },
+    },
+  ],
+  response: {
+    transition: '_success',
+  },
+  on: {
+    _success: {
+      id: 'A A',
+      label: 'A A label',
+      status: 'success',
+      type: 'single',
+      response: {
+        invocations: {},
+      },
+    },
+  },
+};
+
+export const compositeWithComplexTransitions = {
+  id: 'A',
+  label: 'A label',
+  status: 'error',
+  type: 'composite',
+  subtasks: [
+    {
+      id: 'A-A',
+      label: 'A-A label',
+      status: 'error',
+      type: 'single',
+      response: {
+        invocations: {},
+      },
+    },
+    {
+      id: 'A-B',
+      label: 'A-B label',
+      status: 'success',
+      type: 'single',
+      on: {
+        _error: {
+          id: 'A-B A',
+          label: 'A-B A label',
+          status: 'success',
+          type: 'single',
+          response: {
+            invocations: {},
+          },
+        },
+      },
+      response: {
+        invocations: {},
+        transition: '_error',
+      },
+    },
+  ],
+  on: {
+    _success: {
+      id: 'B',
+      label: 'B label',
+      status: 'error',
+      type: 'single',
+      response: {
+        invocations: {},
+      },
+    },
+  },
+  response: {
+    invocations: {},
+    transition: '_success',
+  },
+};
+
+export const singleNodeWithTransitions = {
+  id: 'A',
+  label: 'A label',
+  status: 'success',
+  type: 'single',
+  on: {
+    _success: {
+      id: 'A A',
+      label: 'A A label',
+      status: 'unprocessed',
+      type: 'single',
+    },
+    _error: {
+      id: 'A B',
+      label: 'A B label',
+      status: 'unprocessed',
+      type: 'single',
+    },
+    custom1: {
+      id: 'A C',
+      label: 'A C label',
+      status: 'success',
+      type: 'single',
+      response: {
+        invocations: {},
+      },
+    },
+    custom2: {
+      id: 'A D',
+      label: 'A D label',
+      status: 'unprocessed',
+      type: 'single',
+    },
+  },
+  response: {
+    invocations: {},
+    transition: 'custom1',
+  },
+};
+
+export const compositeWithUnprocessedNodeInSubtask = {
```

Comment:
Which node is unprocessed?

Reply:
The one with id `A-A B` (`root.subtasks[0].on._error`). It's a leaf and the point of the test is to make sure the transition from it to the virtual node is rendered with dashes.